share/models.py

```diff
@@ -958,10 +957,8 @@ def fetch_shared_link_clicked_count(
             date_as_string = "{date}".format(date=date_as_integer)
             date_start = datetime.strptime(date_as_string, "%Y%m%d").date()
             datetime_start = datetime.combine(date_start, datetime.min.time())
-
-            # pst_timezone = pytz.timezone("America/Los_Angeles")
-            # date_start_pst = pst_timezone.localize(datetime_start)
-            date_start_pst = get_timezone_and_datetime_now(datetime_obj=datetime_start)[1]
+            pst_timezone = pytz.timezone("America/Los_Angeles")
+            date_start_pst = pst_timezone.localize(datetime_start)
```

Comment:
Did you mean to roll these changes back?

Reply:
i think i reverted these back to the original code. I didn't think creating a function to replace this logic was necessary because it doesn't occur in other places.